src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Label/LabelHandler.iOS.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UIKit;
+
+namespace Xamarin.Platform.Handlers
+{
+	public partial class LabelHandler : AbstractViewHandler<ILabel, UILabel>
+	{
+		protected override UILabel CreateNativeView() => new UILabel();
+
+		public static void MapText(LabelHandler handler, ILabel label)
+		{
+			ViewHandler.CheckParameters(handler, label);
+
+			handler.TypedNativeView?.UpdateText(label);
+		}
+
+		public static void MapTextColor(LabelHandler handler, ILabel label)
+		{
+			ViewHandler.CheckParameters(handler, label);
+
+			handler.TypedNativeView?.UpdateTextColor(label);
```

Comment:
What about the iOS defaults?

Reply:
I think it is added in the LabelExtensions if color == default, then ColorExtensions.LabelColor

Reply:
Yeah, the iOS default is just black. It's already dealt with in the UpdateTextColor method.

Reply:
but now that's kind of inconsistent right? 

Like we could do this for literally every default. Why even store the defaults at all on the handler?